api/src/functionalTest/java/uk/gov/hmcts/payment/functional/fixture/PaymentFixture.java

```diff
@@ -58,6 +58,75 @@ public static CreditAccountPaymentRequest aPbaPaymentRequest(String amountString
             .build();
     }
 
+    public static CreditAccountPaymentRequest aPbaPaymentRequestForUnspec(String amountString, Service service) {
```

Comment:
Can we combine all these mock request into one dynamic by passing Service Specific SiteIds, we can reduce a good amount of duplicate codes

Reply:
Points noted. We will look into this in the next release.